packages/Debian/bin/make_deb

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -e
+
+usage="$(basename "$0") {x86|arm}
+This script is used to create the Debian package as the
+PostgreSQL Storage Plugin of FogLAMP.
+
+Arguments: 
+ x86 - Build an x86_64 package
+ arm - Build an armv7l package"
+
+## TODO: Use a good way to identify the GIT_ROOT directory
+GIT_ROOT="${HOME}/storage-postgres"
+
+version=`head -1 ${GIT_ROOT}/version`
+BUILD_ROOT="${GIT_ROOT}/packages/Debian/build"
```

Comment:
error if BUILD_ROOT does not already exist:
ususer@us1704:~/storage-postgres$ ./packages/Debian/bin/make_deb x86
The platform is set as x86_64
The package name is foglamp-storage-postgres-00.01-9.6.201608131-x86_64
./packages/Debian/bin/make_deb: line 42: cd: /home/ususer/storage-postgres/packages/Debian/build: No such file or directory

Reply:
Fixed.